core/src/main/java/com/graphhopper/storage/GraphHopperStorage.java

```diff
@@ -65,22 +65,7 @@ public GraphHopperStorage(Directory dir, EncodingManager encodingManager, boolea
         this.dir = dir;
         this.properties = new StorableProperties(dir);
         this.segmentSize = segmentSize;
-        InternalGraphEventListener listener = new InternalGraphEventListener() {
-            @Override
-            public void initStorage() {
-                for (CHGraphImpl cg : chGraphs) {
-                    cg.initStorage();
-                }
-            }
```

Comment:
Probably after some previous refactorings the `initStorage` call did not do anything that we cannot do in the constructor anymore so it is not needed at all.